src/runtime/entitlements-manager-test.js

```diff
@@ -31,6 +31,7 @@ import {
 import {AnalyticsService} from './analytics-service';
```

Comment:
I believe this file tests that analytics service was called with logEvent API.

Since it now calls event manager, this verification should happen on a mock of the event manager.

Reply:
EntitlementsManager is still telling the analytics service to handle the logging.  So it shouldn't test the internal implementation of analytics service by ensuring analytics service passes the event to event manager.  So I don't think any of this needs to change until/unless we circle back and actually have entitlements manager directly send it's event to event manager.

Reply:
You are right. You are only stubbing the deps so that analytics service gets it.